src/main/java/hudson/plugins/jabber/im/transport/JabberIMConnection.java

```diff
@@ -88,7 +65,19 @@
 import org.jxmpp.util.XmppStringUtils;
 import org.springframework.util.Assert;
 
-import sun.security.util.HostnameChecker;
+import hudson.Util;
```

Comment:
It's what happens when you do 'Organize imports' in Eclipse (which is enabled in my default save actions). This removed a number of unused imports.